frontend/integration-tests-cypress/cypress/support/utils.ts

```diff
@@ -0,0 +1,13 @@
+export enum timeouts {
+  default = 4000,
+  pageLoad = 18000,
+}
+export const testName = `test-${Math.random()
```

Comment:
Perhaps this should be a function, so that we can create many test names across the various tests?

Reply:
I don't know that it matters if each set of tests creates its own namespace.  Perhaps we should encourage it (it would affect the run time a bit, however).  Perhaps two functions:

```javascript
let newNamespaceName = () => return `test-${Math.random()... // all that good stuff`
let sharedNamespace = () => call newNamespaceName() once, cache it, always return that value.
```